src/components/Learn-Components/TOC-Learning-Path/toc.style.js

```diff
@@ -77,8 +100,26 @@ const TOCWrapper = styled.div`
             }
         }
     }
-    @media (max-width:767px){
+    
+    @media only screen and (max-width: 991px){
         position: initial;
+        margin-left: .25rem;
```

Comment:
For the alignment, changing this to `-0.5rem` does the trick on my browser at least. You can give it a try though.

Reply:
Got It, It looks good on my browser too. Can you please review @Nikhil-Ladha